### PR TITLE
Markus/remote logging

### PR DESCRIPTION
**Pull request by hemarkus**
Add remote logging from wrapper to master server

### CODE REVIEW

**Comment by hemarkus** [comment]
refs #11

File: qmstr/qmstr-util/remotelog.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	host     string
+	port     int
+	endpoint string
+}
+
+func NewHTTPRemoteLogger(host string, port int, endpoint string) *HTTPRemoteLogWriter {
+	return &HTTPRemoteLogWriter{host, port, endpoint}
+}
+
+func (rlw *HTTPRemoteLogWriter) Write(p []byte) (int, error) {
```
Comment (GiasemiSh) [inline qmstr/qmstr-util/remotelog.go:24]: This method is never used

Comment (hemarkus) [reply qmstr/qmstr-util/remotelog.go:24]: Of course it is. It actually is what makes remote logging work. It is the implementation of io.Writer so the HTTPRemoteLogWriter can be used as a sink for the logger.

Comment (GiasemiSh) [reply qmstr/qmstr-util/remotelog.go:24]: You are right, I got it.

**Comment by GiasemiSh** [comment]
Looks good to me.

**Comment by mirkoboehm** [comment]
LGTM

**Pull request merged**